futures-util/src/stream/try_stream/mod.rs

```diff
@@ -76,6 +78,18 @@ mod try_filter;
 #[allow(unreachable_pub)] // https://github.com/rust-lang/rust/issues/57411
 pub use self::try_filter::TryFilter;
 
+mod try_forward;
```

Comment:
I think this should fix the CI tests.
```suggestion
#[cfg(feature = "sink")]
mod try_forward;
```